index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
To start, I had your project cloned and it seems your project is not complete. I say this because it doesn't follow the Figma design. You are missing some background images and some more `CSS`. Please take a step back to have a look at the project to ensure that it is identical to the Figma design before submitting it for another review.

Your design currently looks like this
![image](https://user-images.githubusercontent.com/29071984/170605419-532ab886-71ba-4569-9d8b-338d8295f98d.png)

And this is how the Figma design looks like
![image](https://user-images.githubusercontent.com/29071984/170605560-806f146a-9f30-4f02-a84f-ddc44cbe61db.png)

It is required that your project looks exactly like the Figma design as possible. Please let's fix that.

Reply:
Thank you so much for the review. But I don't seem to understand the issue with the HTML and CSS. Because I ran it on my browser and inspected it to view the mobile version and the background seem to the there. Kindly shed more light. Thank you

Reply:
@Eternalgratis Oh, I see, please have a look at the relative path of your images then. Also, try it in a different browser and let's have a look.

I see you have made some changes... I am going to check again if the images are displaying.

Reply:
I see it's working now. cheers 🥇 Feel free to ask for another review when you can
![image](https://user-images.githubusercontent.com/29071984/170705640-9602eb43-36db-4294-bac9-b932b19b33c8.png)